masschallenge/django-accelerator
# [AC-6763-c] Office hours interface use smart locations

**Changes Introduced in [AC-6763](https://masschallenge.atlassian.net/browse/AC-6763):**
- Office hours interface uses locations from attached program's program family

**Test:**

_checkout this branch_
- Login as `Demo Admin`
- Masquerade as the user `1032-user@example.com`
- Visit http://localhost:8181/officehours/bos/2019/
- Notice that only locations for the `Boston` program are shown

_checkout development_
- Notice that locations for all programs are shown

Sibling PR: https://github.com/masschallenge/accelerate/pull/2396
